### PR TITLE
Fix `Beatmap Editor Guides` folder

### PR DESCRIPTION
**Pull request by TPGPL**
(´・◡・｀)

### CODE REVIEW

**Comment by TPGPL** [comment]
fuking conflicts? excuse me?

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
 General info
-============
+-------------
 
 Hand this to your favourite QAT.
 
 -   Syntax is as follows:
 
-`[firstOption:firstValue,secondOption:secondValue] and so on, so forth; notice there are no spaces around punctuation marks`
+`[firstOption:firstValue,secondOption:secondValue]` and so on, so forth; notice there are no spaces around punctuation marks
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: L26: Suggestion: add spacing ``[firstOption : firstValue, secondOption: secondValue]`` for readability

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
 
 -   You can split title text into lines using the pipe character, |
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: L28: is that letter I or a ``|``?

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
+Changing Title Text
+=====================
+
 [*Changing the Title Text : The Guide* by: Ekaru](https://osu.ppy.sh/forum/t/14513)
 
 How to get it changed
-=====================
+-------------------------
 
 Sadly, you can't do this yourself. Just ask a QAT to change it. They're the only ones that are able to, so just give them a note of what you need/want, and if it's reasonable enough, you'll get your new title text on your map! Easy, right?!
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: L9: Link QAT from ``People``

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 4]
```diff
@@ -1,33 +1,36 @@
...
 
 Why change?
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: L11: Suspected the grammar here to be too simple. Suggested: ``Why/When do I need to change?``

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 2 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: letter

Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: or no wait

Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: it's a pipe `|`

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 1 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: it's syntax so i shouldnt change anything?

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md [hunk 5]
```diff
@@ -13,42 +16,49 @@ Make a new document. The template skin is 666x666, and while it's safe to go ove
...
 ### Select your circle
 
 Use the ellipse tool (shortcut is "e") and make a circle on the entire document. Create a new layer and fill it with a color.
-![](MPCS_02.png "fig:MPCS_02.png")
+
+![](MPCS_02.png)
 
 ### Shrink the selection
 
 Then shrink the selection by going to Select -&gt; Shrink. Fore more accurate circles, simply redraw them with the ellipse tool, as "shrink" is only good for one or two circles.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: `Select -> Shrink`

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md [hunk 6]
```diff
@@ -13,42 +16,49 @@ Make a new document. The template skin is 666x666, and while it's safe to go ove
...
-![](MPCS_03.png "fig:MPCS_03.png")
+
+![](MPCS_03.png)
 
 I chose 15, but you can choose however much you want. Create a new layer and fill the smaller selection with a different color so you can see it.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: ASG: point of view
`here chose 15,...`

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md [hunk 7]
```diff
@@ -13,42 +16,49 @@ Make a new document. The template skin is 666x666, and while it's safe to go ove
...
 ### Copying the image
 
 And copy/paste the selection from the image you want to copy:
-![](MPCS_06.png "fig:MPCS_06.png")
+
+![](MPCS_06.png)
 
 ### Set canvas size
 
 Finally, you can add a design in the middle, or recolor the edges. Make sure you reset the canvas size to 666x666 by going to Image -&gt; Canvas Size, and center the image so you have a 1 pixel border around the image.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: `Image -> Canvas Size`

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md [hunk 8]
```diff
@@ -59,31 +69,36 @@ If you want your spinner to have the same background as the actual map, it can b
...
 ### Use the channels tab to turn off editing the alpha channel
 
 Technically, you are done, but if you want to make the background "fill in" with a background spinner meter, then you can use the template skin, or make it yourself. Bring in the template spinner-meter. To recolor it, turn off the alpha channel by selecting it, so you don't modify the transparency at all. Then use the bucket fill and fill the entire area with a gray color or whatever color you want (I used black).
-<img src="MPCS_11.png" title="fig:MPCS_11.png" alt="MPCS_11.png" width="600" />
+
+![](MPCS_11.png)
 
 ### Copy/Paste into new layer, then save your spinner meter
 
 Once you have the color, select the entire alpha channel again (right click on the alpha channel -&gt; channel to selection) copy and paste the spinner background so you have something to fill in for your spinner meter.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: too

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md [hunk 9]
```diff
@@ -59,31 +69,36 @@ If you want your spinner to have the same background as the actual map, it can b
...
 ### Turn down the opacity until you get a color your like
 
 To add some more detail, you can turn down the opacity on the black image.
-![](MPCS_13.png "fig:MPCS_13.png")
+
+![](MPCS_13.png)
 
 ### The fully spun spinner background and meter
 
 Then you can see how it looks filled in by turning on the old layer. Note it has a little bit of an outline. If you want to change the size of that outline, you can use the select - grow or shrink tool before you copy/paste to make the spinner meter.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: style of this article:
select -**>** grow

File: wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md (hunk 6 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: where?

Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: nevermind

Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: L33: `I chose 15`
it's the review line

Comment (kj415j45) [inline wiki/Beatmap_Editor_Guides/Making_Properly_Centered_Spinners/en.md]: ~~wiki is contributors' work, not someone's~~

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 10]
```diff
@@ -1,33 +1,36 @@
...
 How to get it changed
-=====================
+-------------------------
 
-Sadly, you can't do this yourself. Just ask a QAT to change it. They're the only ones that are able to, so just give them a note of what you need/want, and if it's reasonable enough, you'll get your new title text on your map! Easy, right?!
+Sadly, you can't do this yourself. Just ask a [QAT](/wiki/People/Quality_Assurance_Team) to change it. They're the only ones that are able to, so just give them a note of what you need/want, and if it's reasonable enough, you'll get your new title text on your map! Easy, right?!
 
-Why change?
-===========
+Why do I need to change?
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: "need" = "required"
(also, this kinda doesn't make sense) change to `Why should I change it?` or something similar

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 11]
```diff
@@ -1,33 +1,36 @@
...
+------------
 
 If used correctly, it can add a certain effect (in a good way!) to a map. It changes the presentation of your map in an effective manner if used correctly by changing the elements and the time of the appearance of the title. If you're a storyboarder, this might be a very huge benefit to you!
 
 -   An effective example would be [Shiirn's Solar map](http://osu.ppy.sh/s/33483)
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: don't be vague with the link text (and use https)
use `[Noisestorm - Solar (Shiirn)](http://osu.ppy.sh/s/33483)`

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 12]
```diff
@@ -1,33 +1,36 @@
...
 -   You can split the title if it's too long for a single line: [example](https://osu.ppy.sh/s/1391), [another one](https://osu.ppy.sh/s/57560)
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: ditto, use `[IOSYS - Kanbu de Todomatte Sugu Tokeru ~ Kyouki no Udongein (DJPop)](https://osu.ppy.sh/s/1391)` and `[Amane - Yume goro mo, Kinou no Koto -Rolling Contact Remix- (TicClick)](https://osu.ppy.sh/s/57560)`

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 13]
```diff
@@ -1,33 +1,36 @@
...
 
 General info
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: Title case, I guess `General Info`

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 14]
```diff
@@ -1,33 +1,36 @@
...
-============
+-------------
 
 Hand this to your favourite QAT.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: A figure of speech but change to `PM (private message) this to your favorite QAT member.`?

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: I would agree (with TPGPL), don't add spacing, otherwise it would imply that one should add the spaces

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 15]
```diff
@@ -60,7 +63,7 @@ How long the text stays on the screen, including the fade in and out.
 Font size in points.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: add this link to explain what points are `[points](https://en.wikipedia.org/wiki/Point_(typography))`

tl;dr its the smallest unit of measure that is used for the height of a font (it equates to 1/72 of an inch, but converting this to pixels will vary as screen resolutions follows)

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 16]
```diff
@@ -1,10 +1,13 @@
+Changing the Map's Artist or Title
+===================================
+
 [*Full Submission with Title/Artist Change -- Mistakes* by NoHitter (hijacked by peppy)](https://osu.ppy.sh/forum/t/24443)
 
 This is until osz2 will be implemented.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: does this still hold true to today?

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 17]
```diff
@@ -1,10 +1,13 @@
...
 
 When changing the **title or artist** in a map, a problem may occur in which you may not be able to submit it without making a duplicate map.
 
-<span style="color:#FF4000;">**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**</span>
+**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: no need to CAPS THIS ALL OUT, I GUESS..
`**Do not submit if the editor "web" menu is disabled. Your map will duplicate.**`

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 18]
```diff
@@ -15,7 +18,7 @@ Easy Method (don't have any local changes):
 3.  Go into editor. Make sure the "Web" menu is enabled (not greyed out).
 4.  Change the Artist/Title info.
 5.  **Do NOT exit the editor.**
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: change line to `5.  Do **not** exit the editor.`

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 19]
```diff
@@ -15,7 +18,7 @@ Easy Method (don't have any local changes):
...
-6.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+6.  Use **File > Open...** to change the Artist/Title in other difficulties.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: idk, but since these are buttons, ``6.  Use `File` > `Open...` to change the Artist/Title in other difficulties.``

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 20]
```diff
@@ -26,9 +29,9 @@ Hard Method (keep local changes):
 4.  Go into editor. Make sure the "Web" menu is enabled (not greyed out).
 5.  Change the Artist/Title info.
 6.  **Do NOT exit the editor.**
-7.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+7.  Use **File > Open...** to change the Artist/Title in other difficulties.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: idk, but since these are buttons, ``6.  Use `File` > `Open...` to change the Artist/Title in other difficulties.``

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 21]
```diff
@@ -26,9 +29,9 @@ Hard Method (keep local changes):
...
 8.  Full Submit.
 
 If you have any duplicated maps, please contact a BAT for resolution.
 
-<span style="color:#FF4000;">**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**</span>
+**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: `**Do not submit if the editor "web" menu is disabled. Your map will duplicate.**`

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 22]
```diff
@@ -10,12 +13,12 @@ There are 2 types of compression. **Lossy** and **Lossless** compression.
 Zip
 ---
 
-The \*.osz package is actually a \*.zip file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
+The `*.osz` package is actually a `*.zip` file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: remove the `*` from the file extensions `` `.osz` `` and `` `.zip` ``

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 23]
```diff
@@ -10,12 +13,12 @@ There are 2 types of compression. **Lossy** and **Lossless** compression.
...
 
 Video
 -----
 
-Recommended reading: [Encoding Video](Encoding_Video "wikilink").
+Recommended reading: [Encoding Video](/wiki/Beatmap_Editor_Guides/Compressing_Videos).
 
 Videos are usually where people go wrong. There are thousands of video codecs out there, and *believe it or not, YouTube is usually a terrible place to get videos*. For the best quality, you should always downsize from a high resolution source (720p YouTube videos are fine for this).
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: line 27: I suggest to just link to the audio guide and delete the content of this section instead of regurgitating similar parts again

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 24]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
 Images
 ------
 
-There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use \*.png and \*.jpg compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
+There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use `*.png` and `*.jpg` compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: remove `*` from file ext, `` `.png` `` and `` `.jpg` ``

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 25]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
...
 
-[Gimp](http://www.gimp.org/) actually does a good job with \*.jpg files.
+[Gimp](http://www.gimp.org/) actually does a good job with `*.jpg` files.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: ditto, `` `.jpg` ``

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 26]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
...
+There are four important things to check: Optimize, Progressive, subsampling, and quality. Optimize is just a way to compress the `*.jpg` without losing quality and should always be checked. Progressive is a method to make jpgs "fade in" when downloaded from the internet. It has a side effect of making the image smaller so long as the image is over 20 KB. Subsampling is a way to improve quality or compression. No subsampling (1x1) will make text and lines much clearer at the cost of space. 2x2 subsampling does very well for photographs which do not have distinct lines (like text). Quality is the easiest way to keep the image looking good. I use 95, since the gains after 95 aren't really worthwhile.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: ditto, `` `.jpg` ``

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 27]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
...
 
-Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [Pngout](http://advsys.net/ken/utils.htm) are two of the best tools to compress \*.png files. Optipng can find the optimum settings for the \*.png, and pngout re-compresses the image using \*.kzip. Often times just using pngout will reduce the image size by 20%.
+Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [Pngout](http://advsys.net/ken/utils.htm) are two of the best tools to compress `*.png` files. Optipng can find the optimum settings for the `*.png`, and pngout re-compresses the image using `*.kzip`. Often times just using pngout will reduce the image size by 20%.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: ditto, `` `.png` ``, `` `.png` ``, and `` `.kzip` ``

File: wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md [hunk 28]
```diff
@@ -1,3 +1,6 @@
+Compressing Videos
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md]: hmm, I guess I'll look into this article and might do a rewrite later (probably in the next month or so)

File: wiki/Beatmap_Editor_Guides/osu!mania_Mapping_Guide/en.md [hunk 29]
```diff
@@ -1,21 +1,22 @@
+osu!mania Mapping Guide
+=========================
+
 Prologue
-========
+---------
 
-What is Mania?
---------------
+### What is Mania?
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/osu!mania_Mapping_Guide/en.md]: `_See also [Mania](/wiki/Game_Modes/osu!mania)_`?

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: I would agree (with TPGPL), don't add spacing, otherwise it would imply that one should add the spaces

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 14 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: going with `Send a PM (private message) this to your favorite QAT member.`

Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: `Send a PM (private message) to your favorite QAT member.`

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 15 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: used reference link cuz of problems

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md (hunk 22 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: why?

Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: nvm

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md (hunk 23 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: Audio Editing?

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md (hunk 22 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: idk, when I wrote the ASG, someone (I don't remember who but they had) suggested that we don't prefix file ext with an `*` (and so that became a thing)

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md (hunk 23 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: yeah, to `/wiki/Beatmap_Editor_Guides/Audio_Editing/` should do

Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: hmm, now I don't know if you should really remove the content of that section now ¯\\\_(ツ)\_\/¯

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md (hunk 1 above)
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: It would improve how it would look though unless the codeblock parser is designed to add a space in-between

File: wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md [hunk 30]
```diff
@@ -1,33 +1,36 @@
...
+-   An effective example would be [Noisestorm - Solar (Shiirn)](https://osu.ppy.sh/s/33483)
+-   You can split the title if it's too long for a single line: [IOSYS - Kanbu de Todomatte Sugu Tokeru ~ Kyouki no Udongein (DJPop)](https://osu.ppy.sh/s/1391), [Amane - Yume goro mo, Kinou no Koto -Rolling Contact Remix- (TicClick)](https://osu.ppy.sh/s/57560)
 
-General info
-============
+General Info
+-------------
 
-Hand this to your favourite QAT.
+Send a PM (private message) with this to your favorite QAT member.
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_Title_Text/en.md]: L22: I suggest redoing this. Suggested ``Send a PM (...) with this to a QAT member``

File: wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md [hunk 31]
```diff
@@ -26,9 +27,9 @@ Hard Method (keep local changes):
...
 5.  Change the Artist/Title info.
 6.  **Do NOT exit the editor.**
-7.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+7.  Use `File` > `Open...` to change the Artist/Title in other difficulties.
 8.  Full Submit.
 
 If you have any duplicated maps, please contact a BAT for resolution.
 
-<span style="color:#FF4000;">**DO NOT SUBMIT IF THE EDITOR "WEB" MENU IS DISABLED. YOUR MAP WILL DUPLICATE.**</span>
+**Do not submit if the editor "web" menu is disabled. Your map will duplicate.**
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md]: 35: Revise. Suggested ``Do not submit if the editor "web" menu is disabled. Else, your map will be duplicated.

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 32]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
 Images
 ------
 
-There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use \*.png and \*.jpg compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
+There are 2 major image formats which make up most of the images you find online: jpg and png. Png uses lossless compression which makes it ideal for small and or simple images. Complex images are better suited for jpg. The majority of image editing programs however do not naturally use `.png` and `.jpg` compression to the most of their ability however, leaving you with 4 KB pngs of 1 color, which could be compressed up to 400 times smaller.
 
-[Gimp](http://www.gimp.org/) actually does a good job with \*.jpg files.
+[Gimp](http://www.gimp.org/) actually does a good job with `.jpg` files.
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: 43: change Gimp to GIMP. GIMP is an acronym that stands for GNU Image Manipulation Program.

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md [hunk 33]
```diff
@@ -35,15 +38,15 @@ Osu accepts 2 major file formats: mp3 and ogg vorbis. Vorbis has a clear advanta
...
 
-Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [Pngout](http://advsys.net/ken/utils.htm) are two of the best tools to compress \*.png files. Optipng can find the optimum settings for the \*.png, and pngout re-compresses the image using \*.kzip. Often times just using pngout will reduce the image size by 20%.
+Pngs are actually much simpler to compress, but GIMP does not do it very well. [Optipng](http://optipng.sourceforge.net/) and [Pngout](http://advsys.net/ken/utils.htm) are two of the best tools to compress `.png` files. Optipng can find the optimum settings for the `.png`, and pngout re-compresses the image using `.kzip`. Often times just using pngout will reduce the image size by 20%.
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: L49: change Png to PNG. PNG is an acronym that stands for Portable Network Format.

File: wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md [hunk 34]
```diff
@@ -14,15 +17,15 @@ Removing audio
 
 3) If the following shows up because the video is h.264 (h.264 is the best right now):
 
-![](CV_01.png "CV_01.png")
+   a) Press either (no is better unless the program crashes).
 
-a) Press either (no is better unless the program crashes).
+![](CV_01.png "CV_01.png")
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md]: L22: Add a text inside ``[]`` to indicate which should be there when the image doesn't load.

File: wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md [hunk 35]
```diff
@@ -14,15 +17,15 @@ Removing audio
...
 
-4) Select Audio -&gt; Main Track
+4) Select Audio -> Main Track
 
 ![](CV_02.png "CV_02.png")
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md]: L26: Add a text inside ``[]`` to indicate which should be there when the image doesn't load.

File: wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md [hunk 36]
```diff
@@ -14,15 +17,15 @@ Removing audio
...
 
-5) Select Audio Source -&gt; None
+5) Select Audio Source -> None
 
 ![](CV_03.png "CV_03.png")
```
Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md]: L30: Add a text inside ``[]`` to indicate which should be there when the image doesn't load.

File: wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md (hunk 33 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: *Portable Network Graphics ¯\\\_(ツ)\_/¯

Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: @MegaApplePi you do realize I edited it already lol

Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: *sigh*, GitHub, Y U NO UPDATE?

Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: the page doesn't update when you edit comments on here ¯\\\_(ツ)\_/¯

Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: You know Chrome and Chromium updates it properly in my end without refreshing 🤷‍♂️

Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: even on the receiving end? (I'm using Chrome Canary btw)

Comment (sr229) [inline wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md]: yeah, but I prefer Chromium since it does auto-update
